.github/workflows/conda-package.yml

```diff
@@ -322,7 +406,130 @@ jobs:
+          dir ${{ env.extracted-pkg-path }}
+
+      - name: Setup miniconda
+        uses: conda-incubator/setup-miniconda@d2e6a045a86077fb6cad6f5adf368e9076ddaa8d # v3.1.0
+        with:
+          miniforge-version: latest
+          use-mamba: 'true'
+          channels: conda-forge
+          conda-remove-defaults: 'true'
+          python-version: ${{ matrix.python }}
+          activate-environment: ${{ env.TEST_ENV_NAME }}
+
+      - name: Store conda paths as envs
+        run: |
+          @echo on
+          (echo CONDA_LIB_PATH=%CONDA_PREFIX%\Library\lib\) >> %GITHUB_ENV%
+          (echo CONDA_LIB_BIN_PATH=%CONDA_PREFIX%\Library\bin\) >> %GITHUB_ENV%
+
+      - name: Install conda-index
+        run: mamba install conda-index=${{ env.CONDA_INDEX_VERSION }}
+
+      - name: Create conda channel
+        run: |
+          @echo on
+          python -m conda_index ${{ env.channel-path }}
+
+      - name: Test conda channel
+        run: |
+          @echo on
+          mamba search ${{ env.PACKAGE_NAME }} -c ${{ env.channel-path }} --override-channels --info --json > ${{ env.ver-json-path }}
+
+      - name: Dump version.json
+        run: more ${{ env.ver-json-path }}
+
+      - name: Get package version
+        run: |
+          @echo on
+          set "SCRIPT=${{ env.VER_SCRIPT1 }} ${{ env.VER_SCRIPT2 }}"
+          FOR /F "tokens=* USEBACKQ" %%F IN (`python -c "%SCRIPT%"`) DO (
+            set PACKAGE_VERSION=%%F
+          )
+          echo PACKAGE_VERSION: %PACKAGE_VERSION%
+          (echo PACKAGE_VERSION=%PACKAGE_VERSION%) >> %GITHUB_ENV%
+
+      - name: Install dpnp
+        run: |
+          @echo on
+          mamba install ${{ env.PACKAGE_NAME }}=${{ env.PACKAGE_VERSION }} pytest pytest-xdist python=${{ matrix.python }} ${{ env.TEST_CHANNELS }}
+        env:
+          TEST_CHANNELS: '-c ${{ env.channel-path }} ${{ env.CHANNELS }}'
+          MAMBA_NO_LOW_SPEED_LIMIT: 1
+
+      - name: List installed packages
+        run: mamba list
+
+      - name: Activate OCL CPU RT
+        shell: pwsh
+        run: |
+          $script_path="$env:CONDA_PREFIX\Scripts\set-intel-ocl-icd-registry.ps1"
+          if (Test-Path $script_path) {
+            &$script_path
+          } else {
+            Write-Warning "File $script_path was NOT found!"
+          }
+          # Check the variable assisting OpenCL CPU driver to find TBB DLLs which are not located where it expects them by default
+          $cl_cfg="$env:CONDA_PREFIX\Library\lib\cl.cfg"
+          Get-Content -Tail 5 -Path $cl_cfg
+
+      - name: Smoke test
+        run: |
+          python -c "import dpnp, dpctl; dpctl.lsplatform()"
+          python -c "import dpnp; print(dpnp.__version__)"
+
+      - name: Run full tests
+        if: matrix.python == '3.12'
+        env:
+          DPNP_TEST_ALL_INT_TYPES: 1
+        run: |
+          pytest -n auto -ra --pyargs ${{ env.PACKAGE_NAME }}.tests
```

Comment:
Would it be better to move that extra step to `test_windows` job, because all the other content of new job is identical (the same for `test_linux`)?

Reply:
I think it is better to keep that way for now and then decide what to do next.
The main idea is to see if only integer dtype tests fails or not.
Having tests with both DPNP_TEST_ALL_INT_TYPES=1 and DPNP_TEST_ALL_INT_TYPES=0 in the same job would complicate this task